tests/Integration/Resources/ProxyProvider/Test_DIRACCAProxyProvider.py

```diff
@@ -12,11 +12,9 @@
 from DIRAC.Core.Security.X509Chain import X509Chain  # pylint: disable=import-error
 from DIRAC.Resources.ProxyProvider.ProxyProviderFactory import ProxyProviderFactory
 
-# For Jenkins
-for f in ['', 'TestCode', os.environ['DIRAC']]:
-  certsPath = os.path.join(f, 'DIRAC/Core/Security/test/certs')
-  if os.path.exists(certsPath):
-    break
+thisPath = os.path.dirname(os.path.abspath(__file__)).split('/')
+rootPath = thisPath[:len(thisPath) - 4]
+certsPath = os.path.join('/'.join(rootPath), 'Core/Security/test/certs')
```

Comment:
Are you trying to find the root of DIRAC ?

Reply:
I need here path to ..Core/Security/test/certs/

Reply:
An absolute path

Reply:
why not just `os.path.join(os.environ['DIRAC'],'Core/Security/test/certs')` ?

Reply:
this way is also possible